tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/CustomTestMethodAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="CustomTestMethodAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Reflection;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2;
+
+internal class CustomTestMethodAttribute
```

Comment:
Is there a reason you named this as an "Attribute"? Could it be `CustomTestMethodExecutor`?

Reply:
Or `SkipTestMethodExecutor`?

Reply:
done in [0554177](https://github.com/DataDog/dd-trace-dotnet/pull/3341/commits/0554177875a4b6a3984eb3d833369811400eedba)